kong/manifest.json

```diff
@@ -33,6 +33,7 @@
     "monitors": {},
     "dashboards": {
       "Kong Overview": "assets/dashboards/kong_overview.json",
+      "Kong Overview Legacy": "assets/dashboards/kong_overview_legacy.json",
```

Comment:
What's the difference between `kong_overview_legacy.json` and `kong_dashboard.json`? Both seem to be legacy?

Reply:
There isn't much difference between them and after chatting with Ofek earlier today, I'll be removing kong_dashboard.json. I was going to add a new PR to remove that separately, but I also wanted to discuss in standup in case we need to worry about customers using that dashboard.